sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -203,6 +203,12 @@ impl Transport for CW310 {
             usb.pin_set_state(CW310::PIN_SW_STRAP0, true)?;
             usb.pin_set_state(CW310::PIN_SW_STRAP1, true)?;
             usb.pin_set_state(CW310::PIN_SW_STRAP2, true)?;
+            usb.pin_set_state(CW310::PIN_TAP_STRAP1, true)?;
+            usb.pin_set_state(CW310::PIN_SRST, false)?;
+            std::thread::sleep(fpga_program.rom_reset_pulse);
+            usb.pin_set_state(CW310::PIN_SRST, true)?;
```

Comment:
I don't think this reset sequence is needed here is it? since bootstrap will issue this reset sequence [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/opentitanlib/src/bootstrap/mod.rs;drc=a4008df8c696af2a514e9964a9e5996c21f4de0e;l=168)

Reply:
For a chip that has 2 reset pins and 5 strapping pins, I'm not confident in that bootstrapping, but I could try it without a reset and if it works and I need to restart CI I'd be happy to push the change.

Reply:
Yep, you're right